test/manual/tls_support.test.ts

```diff
@@ -283,9 +280,17 @@ describe('TLS Support', function () {
   });
 });
 
-function makeConnectionTest(connectionString: string, clientOptions?: MongoClientOptions) {
+function makeConnectionTest(
+  connectionString: string,
+  uriOptions: Record<string, any>,
+  clientOptions?: MongoClientOptions
+) {
+  const uri = new ConnectionString(connectionString);
```

Comment:
Relying on the ConnectionString package to build the URI for us fixed these tests.